src/pages/library/DataBrowserPreview.js

```diff
@@ -130,8 +131,15 @@ const DataBrowserPreview = ({ id }) => {
     !snapshot ?
       centeredSpinner() :
       div({ style: { padding: 20 } }, [
-        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'top', width: '100%', lineHeight: '26px' } }, [
-          h1([snapshot['dct:title']])
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'top', justifyContent: 'space-between', width: '100%', lineHeight: '26px' } }, [
+          h1([snapshot['dct:title']]),
+          h(Link, {
+            onClick: () => { Nav.goToPath('library-details', { id: Nav.getCurrentRoute().params.id }) },
```

Comment:
Toyed with the idea of making this `Nav.history.goBack()`, but decided to go with a new path instead just in case the user got to the page by copy-pasting the link from somewhere else

Reply:
This should get the href from the link if you're going to a known path, rather than the onClick.